src/transformers/models/esm/modeling_esm.py

```diff
@@ -0,0 +1,1408 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
```

Comment:
Should the Facebook copyright be added here?

Reply:
Done!